.changeset/cuddly-kids-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': patch
+---
+
+Fix for issues related to SVG loading
```

Comment:
I _think_ we don't need a changelog entry for this because it only affects the storybook site, not the npm package

Reply:
All righty, removed the changeset!